assets/src/edit-story/components/library/panes/text/textSets/loadTextSets.js

```diff
@@ -55,8 +55,8 @@ async function loadTextSet(name) {
         ...e,
         // Offset elements so the text set's
         // default position is (0,0)
-        previewOffsetX: e.x - minMax.minX,
-        previewOffsetY: e.y - minMax.minY,
+        normalizedOffsetX: e.x - minMax.minX,
+        normalizedOffsetY: e.y - minMax.minY,
```

Comment:
I renamed `previewOffset` to `normalizedOffset` since these values are simply the X, Y values that'll place the text set (while maintaining correct layout) at a (0, 0) position.  Using this value, it was easy to calculate the correct offset based on the mouse position during drag and drop.